frontend/internal-packages/jobs/src/trigger/chatJobs.ts

```diff
@@ -0,0 +1,41 @@
+import { createSupabaseRepositories, processChatMessage } from '@liam-hq/agent'
+import type { ChatProcessorParams } from '@liam-hq/agent'
+import { logger, task } from '@trigger.dev/sdk/v3'
+import { createClient } from '../libs/supabase'
+
+// Define type excluding repositories
+type ChatJobPayload = Omit<ChatProcessorParams, 'repositories'>
+
+export const processChatTask = task({
+  id: 'process-chat-message',
+  run: async (payload: ChatJobPayload) => {
+    logger.log('Starting chat processing job:', {
+      buildingSchemaId: payload.buildingSchemaId,
+      messageLength: payload.message.length,
+    })
+
+    try {
```

Comment:
Is try-catch really necessary?

Reply:
It was an unnecessary try-catch🙏. I checked and it seemed to terminate the error without throwing an error.
https://github.com/liam-hq/liam/pull/1993/commits/840462a95cf9305e476f3888e64d6902b2c69e2c